terraform/main.tf

```diff
@@ -15,3 +15,22 @@ module "ec2-instance" {
   AWS_INSTANCE_TYPE = var.AWS_INSTANCE_TYPE
 }
 
+module "alb" {
+  source = "./modules/alb"
+  vpc-id = module.vpc.vpc-id
+  alb-security-group-id = module.vpc.alb-security-group-id
+  public-subnets = module.vpc.public-subnets
+  public-subnet-id-1 = module.vpc.public-subnet-id-1
+  public-subnet-id-2 = module.vpc.public-subnet-id-2
+  autoscaling-group-id = module.autoscaling.autoscaling-group-id
+}
+module "autoscaling" {
+  source = "./modules/autoscaling"
+  KEYPAIR_NAME = var.KEYPAIR_NAME
+  AWS_REGION     = var.AWS_REGION
+  AWS_INSTANCE_TYPE = var.AWS_INSTANCE_TYPE
+  instance-security-group-id = module.vpc.instance-security-group-id
```

Comment:
What if we need to attach multiple security groups? This should be an array

Reply:
if we have more two security groups then array can be used.for single value terraform through an error